samples/SocketsSample/project.json

```diff
@@ -55,7 +55,7 @@
   },
 
   "scripts": {
-    "precompile": [ "tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
+    "precompile": [ "../../bin/nodejs/node_modules/.bin/tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
```

Comment:
won't this break in VS though where tsc is installed globally?
Also with VS - not sure how to enforce tsc 2.0